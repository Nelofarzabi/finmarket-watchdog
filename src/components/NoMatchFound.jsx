import React from 'react';

const NoMatchFound = () => (
  <div className="flex justify-center min-h-screen h-full pt-10">
    <h2 className=" text-3xl text-[#96B6C5]">Sorry, No Match Was Found...</h2>
  </div>
);

export default NoMatchFound;
