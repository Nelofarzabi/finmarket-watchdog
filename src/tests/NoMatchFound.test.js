import React from 'react';
import { render } from '@testing-library/react';
import NoMatchFound from '../components/NoMatchFound';

describe('NoMatchFound', () => {
  test('renders with the provided message', () => {
    const { getByText } = render(<NoMatchFound message="Sorry, No Match Was Found" />);

    // Check if the component renders the provided message
    expect(getByText('Sorry, No Match Was Found')).toBeInTheDocument();
  });

  test('matches snapshot', () => {
    const { container } = render(<NoMatchFound message="Sorry, No Match Was Found" />);

    // Check if the rendered component matches the snapshot
    expect(container).toMatchSnapshot();
  });
});
