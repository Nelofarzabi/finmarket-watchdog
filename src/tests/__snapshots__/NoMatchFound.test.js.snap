// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatchFound matches snapshot 1`] = `
<div>
  <div
    class="flex justify-center min-h-screen h-full pt-10"
  >
    <h2
      class=" text-3xl text-[#96B6C5] text-center"
    >
      Sorry, No Match Was Found
    </h2>
  </div>
</div>
`;
